File: CPP/destructor_employee.cpp
#include<iostream>
using namespace std;
class employee
{
public:
		employee()
	{
		cout<<"constructor called"<<endl;
    }
    
    ~employee()
    {
	cout<<"destructor called"<<endl;
    }
};
int main()
{
	employee e1;
	employee e2;
	return 0;
}

File: C#/destructor_employee.cs
using System;
namespace program
{
    class Employee
    {
        public Employee()
        {
            Console.WriteLine("constructor called");
        }
        ~Employee()
        {
            Console.WriteLine("destructor called");
        }
    }
    class program
    {
        static void Main()
        {
            Employee emp = new Employee();
            Console.ReadKey();
            Employee emp2 = new Employee();
            Console.ReadKey();
        }
    }
}